release/triggers/bundle-release/development/BUNDLE_NUMBER

```diff
@@ -1 +1 @@
-67
+68
```

Comment:
I believe this version is wrong. We need a new bundle number across all releases so this would be the increment of release v0.20.1

https://github.com/aws/eks-anywhere/blob/release-0.20/release/triggers/bundle-release/development/BUNDLE_NUMBER